### DISCUSSION TITLE
Use Tab with Next.js Link?

### DISCUSSION THREAD

**Discussion by ptrhck** [chakra-ui/chakra-ui#3937] [Q&A]
Is it a good idea to use the [Tabs](https://chakra-ui.com/docs/disclosure/tabs) components to change the route with Next.js? The examples look like it should be used on the same page. If otherwise, how could this be achieved?

**Comment by bruno-fialho** [comment]
This is how I'm doing this:
```ts
<TabItems
    name="Moviments"
    onClick={() => {
        onClose();

        router.push({
            pathname: `/consults/enterprises/${encodeURIComponent(
                enterpriseCode,
            )}/actions`,
            query: { action: 'moviments' },
        });
    }}
/>
```
In this example, I'm just changing the query between tabs...
